source/install/docker-local-machine.rst

```diff
@@ -68,6 +68,31 @@ Ubuntu
 3. When docker is done fetching the image, open ``http://localhost:8065/``
    in your browser.
 
+Fedora
+^^^^^^
+
+1. Follow the instructions at https://docs.docker.com/engine/installation/linux/fedora/ or use the summary below:
+
+   ..  code:: bash
+      
+        sudo dnf -y install dnf-plugins-core
+        sudo dnf config-manager \
+        --add-repo \
+        https://download.docker.com/linux/fedora/docker-ce.repo
+        sudo dnf install docker-ce docker-compose git # Accepting the new docker repository key
+        sudo usermod -aG docker <username>
+        sudo systemctl start docker
+        cd mattermost-docker
+        sudo docker-compose build
+ 
+2. Start the container: 
+
+   .. code:: bash
+      
+       sudo docker-compose up -d
+       
+3. When the startup is complete, visit ``https://localhost/`` in your browser.  
```

Comment:
Ah I see, thanks for clarifying @JeffSchering , seems my images are a bit messed up by the product deployment in docker so ive cleaned it all out and comes up on the 8065 , ill amend the notes now.